benchmarks/partition.bench.ts

```diff
@@ -3,11 +3,11 @@ import { partition as partitionToolkit } from 'es-toolkit';
 import { partition as partitionLodash } from 'lodash';
 
 describe('partition', () => {
-  bench('es-toolkit', () => {
+  bench('es-toolkit/', () => {
```

Comment:
`es-toolkit/partition` ?

Reply:
🦅👀 I fixed it. Thanks! [3020b5d](https://github.com/toss/es-toolkit/pull/75/commits/3020b5d246dd228336b099ccebed61ebe8d9de7b)